### PR TITLE
feat: implicit commit

### PR DESCRIPTION
**Pull request by LUAgam**
Reason:  
  Improve #2075  inner 552
Type:  
  Improve  
Influences：  
  feat: implicit commit

### CODE REVIEW

File: src/main/java/com/actiontech/dble/services/rwsplit/RWSplitQueryHandler.java [hunk 1]
```diff
@@ -83,4 +85,40 @@ public void query(String sql) {
...
+            case RwSplitServerParse.DROP_VIEW:
+            case RwSplitServerParse.INSTALL:
+            case RwSplitServerParse.RENAME:
+            case RwSplitServerParse.UNINSTALL:
+            case RwSplitServerParse.GRANT:
+            case RwSplitServerParse.REVOKE:
+                return rwSplitService -> {
+                    if (session.getService().isTxStart()) {
+                        rwSplitService.setTxStart(false);
+                        rwSplitService.getSession().unbindIfSafe(true);
```
Comment (yanhuqing666) [inline src/main/java/com/actiontech/dble/services/rwsplit/RWSplitQueryHandler.java:115]: need singleTransactionsCount?

Comment (LUAgam) [reply src/main/java/com/actiontech/dble/services/rwsplit/RWSplitQueryHandler.java:115]: Yes, I'll change it

Comment (LUAgam) [reply src/main/java/com/actiontech/dble/services/rwsplit/RWSplitQueryHandler.java:115]: > need singleTransactionsCount?

Yes, I'll change it

**Review by PanternBao — APPROVED** [review]
lgtm

**Pull request merged**